wallet/src/de/schildbach/wallet/ui/transactions/TxResourceMapper.java

```diff
@@ -41,6 +40,15 @@
 
 public class TxResourceMapper {
 
+    private boolean isCoinJoinMixingFee(Transaction tx) {
+        Coin inputsValue = tx.getInputSum();
+        Coin outputsValue = tx.getOutputSum();
+        Coin netValue = inputsValue.subtract(outputsValue);
+        return tx.getInputs().size() == 1 && tx.getOutputs().size() == 1
+                && CoinJoin.isCollateralAmount(inputsValue)
+                && CoinJoin.isCollateralAmount(outputsValue)
+                && CoinJoin.isCollateralAmount(netValue);
+    }
```

Comment:
In a future PR, this should be moved to DashJ so each CoinJoin TX can be identified for purposes of labeling and grouping.